remotecv/worker.py

```diff
@@ -12,24 +12,19 @@
 import logging
 import sys
 from importlib import import_module
-
-from redis import Redis
+from threading import Thread
 
 from remotecv.error_handler import ErrorHandler
-from remotecv.utils import config
+from remotecv.utils import config, redis_client
+from remotecv.healthcheck import HealthCheckHandler
```

Comment:
Nit: `HealthCheckHandler` is imported but unused.

```suggestion
```

Reply:
Removed import!